src/modules/Auth/index.js

```diff
@@ -1,6 +1,6 @@
-import React, { Component } from "react";
-import axios from "axios";
-import Button from "../../components/Button";
+import React, { Component } from 'react';
+// import axios from "axios";
```

Comment:
Try not to push commented code. Remove it if not used.

Reply:
Okay.